CMakeLists.txt

```diff
@@ -42,13 +42,14 @@ set(GFXRECONSTRUCT_PROJECT_VERSION_MAJOR 0)
 set(GFXRECONSTRUCT_PROJECT_VERSION_MINOR 9)
 set(GFXRECONSTRUCT_PROJECT_VERSION_PATCH 8)
 
-set(GFXRECON_PROJECT_VERSION_DESIGNATION "-unknown")
 set(GFXRECON_PROJECT_VERSION_SHA1 "unknown-build-source")
 
 if (NOT DEFINED HEADLESS)
     set(HEADLESS TRUE)
 endif()
 
+option(BUILD_ZSTD_SUPPORT "Build zstd support" OFF)
```

Comment:
@per-mathisen-arm I think I said incorrectly that this was default OFF, so I think you had this right the first time.  If you could please add a commit to this PR with the default being ON, we'll re-run CI.  It might be that you have to commit that to your branch and I'll have to update the PR; not sure how that will work but I'll check on it in the morning.

Reply:
Done. I've also split out the other changes as their own PRs to make this easier to handle. Although the automation PR will depend on this PR, which is less than optimal.